### ISSUE TITLE
Wire full article view page (Jade)

### ISSUE THREAD

**Issue by roman87** [TeamMentor/TM_4_0_Design#194]
This is to create a page that will open in a new tab displaying an article in full view. Page needs to show meta data and copyright. The page is already developed and needs to be wired.

**Comment by DinisCruz** [comment]
I talking this as also meaning 'view the article content on 4.0 UI and not on 3.5 server'

**Comment by DinisCruz** [comment]
@luchia which one is the Jade page that should be used to display article content?

**Comment by luchia** [comment]
I added it to the User directory for you Dinis, it was in the 'soon to be deleted' directory.

**Comment by DinisCruz** [comment]
This is now working, and its with @luchia to improve the css

![image](https://cloud.githubusercontent.com/assets/656739/6037728/948cf2f6-ac4e-11e4-98ae-2578f8db5e70.png)

**Comment by luchia** [comment]
Taking this to finish it off.

**Comment by luchia** [comment]
As per our meeting today, we discussed the fact that articles will at some point need to be reformatted due to having to hack the CSS to make the sizing right. @roman87 said it was more of a long-term goal and task and for the time being that we should use the hack for the beta.

I spent part of my morning getting sizing right on the article content. See below and the referenced commit.

![screen shot 2015-02-05 at 11 24 32 am](https://cloud.githubusercontent.com/assets/4547737/6053324/b7e909a2-ad29-11e4-88ff-7a6857a538d0.png)
![screen shot 2015-02-05 at 11 24 34 am](https://cloud.githubusercontent.com/assets/4547737/6053323/b7e8f480-ad29-11e4-94b3-91518346c018.png)
